.github/workflows/build-resharper-plugin.yml

```diff
@@ -0,0 +1,33 @@
+on:
+  workflow_dispatch:
+    inputs:
+      wave:
+        description: 'ReSharper Wave (e.g. 243 for 2024.3.x)'
+        required: true
+        default: '100'
+        type: string
+jobs:
+  build:
+    runs-on: windows-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
```

Comment:
Please let's bump this to v4 (though we might just enable Dependabot later, and it will bump stuff for us).